stdlib/public/Concurrency/Task.cpp

```diff
@@ -282,21 +290,21 @@ AsyncTaskAndContext swift::swift_task_create_future_f(
     new(allocation) AsyncTask(&taskHeapMetadata, flags,
                               function, initialContext);
 
+  // Initialize task locals fragment.
+  auto taskLocalsFragment = task->localValuesFragment();
+  new (taskLocalsFragment) AsyncTask::TaskLocalValuesFragment();
+  taskLocalsFragment->initializeLinkParent(task, parent);
```

Comment:
FWIW, if this is unconditionally there in the task, there's no reason for it to be a fragment.

Reply:
Yeah, it used to be conditional before but ended up not being.. I'll follow up with removing the fragment 👍